core/daemon/src/main/resources/META-INF/opennms/applicationContext-daemon.xml

```diff
@@ -29,13 +29,26 @@
   <onmsgi:reference id="eventProxy" interface="org.opennms.netmgt.events.api.EventProxy"/>
   <onmsgi:reference id="eventSubscriptionService" interface="org.opennms.netmgt.events.api.EventSubscriptionService"/>
 
+  <!-- TODO: Remove this bean after we stop using serialized object messages -->
```

Comment:
I thought we got rid of the remaining serialized object messages. Where do we still use them?